cmd/tk/main.go

```diff
@@ -54,10 +55,37 @@ func main() {
 		prefixCommands("tk-")...,
 	)
 
+	outputPprofFile := os.Getenv("TK_PPROF_FILE")
```

Comment:
Envvars are usually called `TANKA_*`: https://tanka.dev/env-vars
Also, describe it there

Reply:
I was going to investigate whether it's better to do a pprof per environment, if the results are more useful then. If that's the case then I would move this to the export subcommand and then we can have an argument instead of an env var.

I'll un-draft after a bit more fiddling!

Reply:
Being able to enable it per-env would be awesome. If an env is taking 10ms to evaluate, we don't really care, vs that one environment that takes 2m